src/newWizard/Wizard.stories.tsx

```diff
@@ -32,16 +32,7 @@ function buildStepData(numberOfSteps: number = 1) {
 // Function to create steps UI
 function buildStepsUI() {
     const StepUI = steps.map((step, index) => {
-        return (
-            <WizardStep
-                id={index}
-                key={index}
-                name={step.name}
-                type={step.type}
-                valid={true}
-                complete={true}
-            ></WizardStep>
-        );
+        return <WizardStep id={index} key={index} name={step.name} type={step.type} valid={true} complete={true} />;
```

Comment:
can we revert this change ? seems not required in this PR

Reply:
I tried to revert the change but formatter is doing this change, not me. Unable to revert this change.